src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -668,12 +668,13 @@ def _runtool_hermetic(self, main, tool_name, distribution, input_digest, ctx):
 
     ctx.rsc_jar_file = ClasspathEntry(ctx.rsc_jar_file.path, res.output_directory_digest)
 
-    self.context._scheduler.materialize_directories((
+    _, = self.context._scheduler.product_request(
```

Comment:
Its not an error to ignore the return value in python and it's probably more idiomatic to just kill assignment of thevalue at all.